pkgs/top-level/all-packages.nix

```diff
@@ -23170,7 +23170,6 @@ with pkgs;
       { };
   inherit (nvidiaCtkPackages)
     nvidia-docker
-    nvidia-podman
```

Comment:
This removes the top-level attribute for `nvidia-podman`. I guess we want to deprecate it instead, but did want to take the opportunity to clean up unneeded stuff in case it's possible.

Reply:
Yeah I guess let's put it in `aliases.nix` with a warning to avoid frustrating anyone